tests/unit/amo/reducers/test_collections.js

```diff
@@ -169,8 +171,10 @@ describe(__filename, () => {
         detail: collectionDetail,
       }));
 
+      const fakeCollectionAddon =
+        createFakeCollectionAddon({ addon: { ...fakeAddon, id: 333 }, notes });
```

Comment:
I usually find more readable to write:

```js
const fakeCollectionAddon = createFakeCollectionAddon({
  addon: { ...fakeAddon, id: 333 },
  notes,
});
```

By having one parameter per line, I find it easier to parse the different parameters at a given level (like here `addon` and `note` are the two params given to `createFakeCollectionAddon`).